### ISSUE TITLE
I could cry because of happiness! <3 <3 <3 - Just a big Thank you!

### ISSUE THREAD

**Issue by aldaw** [oliverschwendener/ueli#811]
I was using aflred for years for on the fly translation, then the workflow/plugin went broken because of the api.
And I never had that experience on windows anyways, but i was looking for it and since it was broken even harder again since years.
Now I found this,.. and I wanted to thank you really all for any bit of code and time you have put into your are amazing!
I wish you all the best health to you and your beloved once!

**Comment by oliverschwendener** [comment]
Thanks for your kind words and I'm very happy that this project helps you out in your daily tasks :) Made my day!

**Comment by oliverschwendener** [comment]
Maybe it's also a good opportunity for me to thank everyone who contributed to this project:

- @tkohlmeier 
- @amrbashir 
- @khanhas 
- @Lukentui 
- @vishaluchil 
- @tvollstaedt 
- @sashaweiss 
- @TimB87 
- @BenoitGour 
- @cathaysia 
- @jatisto 
- @themegaphoenix 
- @jondcoleman 
- @ykhrito 
- @sgruetter 
- @Arpit-Babbar 
- @axieum 
- @mukireus 
- @Mister-Kin 
- @usernamehw 
- @pecigonzalo 
- @underthecocotree 
- @weiw11 
- @edoreld 
- @siketyan 
- @mathiasandresen 
- @atereshhuk68 
- @jcklpe 
- @jmlweb 
- @Khoding 
- @vacekj 
- @tmueller 
- @ddarkr 
- @anicetoclucas

**Comment by axieum** [comment]
Thank you @aldaw & @oliverschwendener!

Now is the perfect time for anyone to get involved and make any little contribution - it's Hacktoberfest time! 🥳

https://hacktoberfest.digitalocean.com/

**Comment by NotWearingPants** [comment]
@axieum 
> it's Hacktoberfest time!
Yes but this project is sadly not participating. Of course contributions are always fun whether you get a shirt or not 😋
